sessionctx/variable/tidb_vars.go

```diff
@@ -323,6 +323,10 @@ const (
 	// Note if you want to set `tidb_enforce_mpp` to `true`, you must set `tidb_allow_mpp` to `true` first.
 	TiDBEnforceMPPExecution = "tidb_enforce_mpp"
 
+	// TiFlashMaxThreads TiflashMaxThreads is the maximum number of threads to execute the request which is pushed down to tiflash.
+	// Default value is -1, means it will not be pushed down to tiflash.
+	// If the value is bigger than -1, it will be pushed down to tiflash and used to create db context in tiflash.
+	TiFlashMaxThreads = "tidb_max_tiflash_threads"
```

Comment:
Can we rename to be consistent:
```suggestion
	TiDBMaxTiFlashThreads = "tidb_max_tiflash_threads"
```

Reply:
> Can we rename to be consistent:

done.